numpy/fft/fftpack.py

```diff
@@ -101,6 +102,11 @@ def _unitary(norm):
     return norm is not None
 
 
+def _fft_dispatcher(a, n=None, axis=None, norm=None):
```

Comment:
As far as I can see, in this whole file, there are only two signatures: `(a, n, axis, norm)`, and `(a, s, axes, norm)` - I think in such a well-constructed file, it is a lot clearer to define the two signatures once and use them throughout.